spec/helpers/node_installer_spec.rb

```diff
@@ -2,13 +2,15 @@
 
 describe LanguagePack::Helpers::NodeInstaller do
   describe "#install" do
-    it "should extract a node binary" do
-      Dir.mktmpdir do |dir|
-        Dir.chdir(dir) do
-          installer = LanguagePack::Helpers::NodeInstaller.new
-          installer.install
+    LanguagePack::Base::KNOWN_ARCHITECTURES.each do |arch|
+      it "should extract a node binary on #{arch}" do
+        Dir.mktmpdir do |dir|
+          Dir.chdir(dir) do
+            installer = LanguagePack::Helpers::NodeInstaller.new(arch: "arm64")
```

Comment:
I think this should be: `installer = LanguagePack::Helpers::NodeInstaller.new(arch: arch)` and not hardcode `arm64`

@schneems